Gopkg.toml

```diff
@@ -96,9 +96,7 @@ required = [
 
 [[constraint]]
   name = "github.com/operator-framework/operator-sdk"
-  # The version rule is used for a specific release and the master branch for in between releases.
-  branch = "master" #osdk_branch_annotation
-  # version = "=v0.5.0" #osdk_version_annotation
+  version = "=v0.8.2"
```

Comment:
Based on the comment I wonder where we are with respect to branching? @rthallisey

Reply:
Sure thing. From HCO perspective, this PR just wants to enable the SSP operator to install our metric aggregations rule. In turn, metrics aggregation rules are just `PrometheusRule`s (prometheus operator CRDs). See: https://github.com/MarSik/kubevirt-ssp-operator/blob/master/roles/KubevirtMetricsAggregation/templates/aggregation-rule-vmi-count.yaml.j2

Reply:
We'll cut the 2.1 branch after we have some more features merged.